File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/cake/CakeDetailParam.java
package com.cakk.domain.mysql.dto.param.cake;

import java.util.Set;

import com.cakk.common.enums.CakeDesignCategory;
import com.cakk.domain.mysql.dto.param.tag.TagParam;

public record CakeDetailParam(
	String cakeImageUrl,
	String cakeShopName,
	String shopBio,
	Long cakeShopId,
	Set<CakeDesignCategory> cakeCategories,
	Set<TagParam> tags
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/cake/CakeDetailParam.kt
package com.cakk.domain.mysql.dto.param.cake

import com.cakk.common.enums.CakeDesignCategory
import com.cakk.domain.mysql.dto.param.tag.TagParam

data class CakeDetailParam(
	val cakeImageUrl: String,
	val cakeShopName: String,
	val shopBio: String,
	val cakeShopId: Long,
	val cakeCategories: Set<CakeDesignCategory>,
	val tags: Set<TagParam>
)
